### DISCUSSION TITLE
KONG Gateway / OPA Official Plugin

### DISCUSSION THREAD

**Discussion by anasawad** [open-policy-agent/community#57] [Community]
I'm using the OPA official plugin with KONG API GW and I'm wondering if there's any way to forward the request body from KONG to OPA through the plugin in the "input"?

**Comment by anderseknert** [comment]
Hi @anasawad ! 

I haven't used that plugin myself, but as far as I understood it doesn't currently provide an option to provide the request body as part of the input. The available options/parameters are listed [here](https://docs.konghq.com/hub/kong-inc/opa/#parameters). Since the plugin is maintained by Kong, I'd suggest reaching out to them with your feature request.

**Reply by anderseknert** [reply]
Or did you mean this integration? https://github.com/open-policy-agent/contrib/tree/main/kong_api_authz

**Reply by anasawad** [reply]
@anderseknert , Thanks! I appreciate your guidance.

**Reply by anasawad** [reply]
> Or did you mean this integration? https://github.com/open-policy-agent/contrib/tree/main/kong_api_authz

No, I mean the OPA official plugin on KONG API GW [here](https://docs.konghq.com/hub/kong-inc/opa/)

**Reply by anderseknert** [reply]
Alright, then it's off to Kong I guess :) If you do hear from them, feel free to post any updates here!
